### DISCUSSION TITLE
LDAP support (from command line if possible)

### DISCUSSION THREAD

**Discussion by DeMiro5001** [joomla/joomla-cms#43741] [Q&A]
Hello everyone.
I have managed to install joomla 5 from the command line on a debian 11 server with nginx and it's running fine.
I looked on the documentation about configuring LDAP for joomla 5 but couldn't find it.
So my questions are : is there a way to enable LDAP from cli ? and to configure it ? if yes, how ?
Thanks a lot for all the effort you take in maintaining this fabulous software and thanks again for taking time to read me.
(My main purpose is to make the installation one-click with the most options possible)

**Comment by richard67** [comment]
The LDAP plugin can be found in the plugin group "authentication". It is not enabled on a new installation. The configuration options should not be different to those in Joomla 4, and also not much different to those for Joomla 3 for which there might be more documentation.

Unfortunately the CLI doesn't provide a command to enable a plugin or to change its configuration options.

**Reply by richard67** [reply]
P.S.: The help page of that plugin for Joomla 5.1 is https://help.joomla.org/proxy?keyref=Chunk4x:Extensions_Plugin_Manager_Edit_Authentication_Group/en#Authentication_-_LDAP .
